lib/alertStateEnum.js

```diff
@@ -0,0 +1,9 @@
+const ALERT_STATE = {
+    STARTED: 'Started',
+    WAITING_FOR_REPLY: 'Waiting for reply',
```

Comment:
I think we discussed this a while ago, but if we had visibility into whether a message was delivered (which we currently don't have with SMS), or read/seen (perhaps on the app) - would this set of {alertStates} expand? 

I guess this is more a theoretical question for sometime down the line...

Reply:
Good question. I don't remember us discussing that before. Yes, I could imagine that list expanding as we have more states that we wish to distinguish.